src/main/java/ch/njol/skript/bukkitutil/block/MagicBlockCompat.java

```diff
@@ -72,7 +72,7 @@
 	private class MagicBlockValues extends BlockValues {
 
 		private Material id;
-		short data;
+		private short data;

```

Comment:
Doesn't this cause synthetic accessors to be generated by compiler? They're somewhat problematic before Java 11 (nestmates JVM feature).

Reply:
The itemFlags variable below is also a private primitive... is that also an issue?

Reply:
It's not related, or specific to primitives. It's a global thing for nested classes and private members.

If you are using Eclipse, it will show a warning when a synthetic accessor is needed to access a field. You can also enable a similar warning in the IntelliJ, from File -> Settings -> Inspections, search for "synthetic"

All fields should be package-private (no additional visibility prefix, directly type, name and obviously the semicolon) or have getters/setters and accessed by getters/setters in nested (sub) classes to not generate a synthetic accessor method.

More on synthetic access: https://stackoverflow.com/a/16226833/12308576